src/Serializer.php

```diff
@@ -185,16 +225,20 @@ public static function decodeMetadata(array $metadata)
         if (count($metadata) == 0) {
             return [];
         }
-        $result = [];
+        $result = [
+            'serialized' => [],
+            'unserialized' => []
+        ];
```

Comment:
This looks like a breaking change to me

Reply:
You are right, didn't thought as this method as an external one but you're right, this would break compatibility